drivers/counter/Kconfig.gecko

```diff
@@ -1,17 +1,31 @@
 # Silicon Labs Gecko Counter driver config
 
 # Copyright (c) 2019, Piotr Mienkowski
+# Copyright (c) 2022 T-Mobile USA, Inc.
+#
 # SPDX-License-Identifier: Apache-2.0
 
 config COUNTER_GECKO_RTCC
 	bool "Silicon Labs Gecko Counter (RTCC) driver"
 	default y
-	depends on DT_HAS_SILABS_GECKO_RTCC_ENABLED
+	depends on DT_HAS_SILABS_GECKO_RTCC_ENABLED && SOC_FAMILY_EXX32
 	select SOC_GECKO_CMU
 	select SOC_GECKO_RTCC
 	help
-	  Enable counter driver based on RTCC module for Silicon Labs Gecko
-	  chips.
+	  Enable counter driver based on RTCC module with the RTCC_CTRL
+	  CNTMODE field set to NORMAL mode running at a 30.5 microsecond rate
+	  overflowing in 36.4 hours for Silicon Labs Gecko chips.
+
+config PERSISTENT_GECKO_RTCC
+	bool "Silicon Labs Gecko Time (RTCC) persistence"
+	default n
+	depends on DT_HAS_SILABS_GECKO_RTCC_ENABLED && SOC_FAMILY_EXX32
+	select SOC_GECKO_RMU
+	select SOC_GECKO_CMU
+	select SOC_GECKO_RTCC
```

Comment:
```suggestion
```

Reply:
How else are you suggesting that I enable RMU settings?